src/http-api/routes/files.js

```diff
@@ -6,6 +6,7 @@ module.exports = (server) => {
   const api = server.select('API')
 
   api.route({
+    // TODO fix method
```

Comment:
Yeah, I put it there. Mainly as a reminder that we should talk about setting the method of the endpoints in `http-api-spec` and change them accordingly here.